packages/use-intl/src/core/DateTimeFormatOptions.tsx

```diff
@@ -1,5 +1,7 @@
 // https://github.com/microsoft/TypeScript/issues/35865
 
+import { TimeZone } from "./utils/TimeZones";
```

Comment:
Some nits:
1. You can move this out of `utils` to the same level where e.g. `DateTimeFormatOptions` is defined.
2. Please use a default export/import like in `DateTimeFormatOptions`.
3. Please make sure the file name matches the import name (the singular `TimeZone` is good)